cmd/ef_tests/levm/runner/levm_runner.rs

```diff
@@ -72,31 +72,33 @@ pub fn prepare_vm_for_tx(vector: &TestVector, test: &EFTest) -> Result<VM, EFTes
         store: initial_state.database().unwrap().clone(),
         block_hash,
     });
+
+    let tx = test.transactions.get(vector).unwrap();
```

Comment:
Here instead of using `.unwrap()` you can return an `EFTestRunnerError`

Reply:
https://github.com/lambdaclass/ethrex/pull/1319/commits/8e882bc3b8192c659a3464ce6472e62dc3112dfd